app/parse.py

```diff
@@ -117,18 +118,45 @@ def get_local_tag(tag) -> str:
     return etree.QName(tag).localname
 
 
-def parse_xml(content: bytes):
-    # TODO: Finish ONIX parsing
-    root = etree.fromstring(content)
-    for product in root.findall("{*}Product"):
-        isbn_10_row = product.find(
-            "{*}ProductIdentifier/[{*}ProductIDType='02']/{*}IDValue"
-        )
-        isbn_10 = isbn_10_row.text if isbn_10_row is not None else None
-        print("ISBN 10: ", isbn_10)
-
-        isbn_13_row = product.find(
-            "{*}ProductIdentifier/[{*}ProductIDType='15']/{*}IDValue"
-        )
-        isbn_13 = isbn_13_row.text if isbn_13_row is not None else None
-        print("ISBN 13: ", isbn_13)
+def parse_xml(content: bytes) -> list[Incomplete]:
+    print("parsing XML...")
```

Comment:
Since this is complete and working now you should probably delete the last 2 print statements here.